packages/components/src/range-control/input-range.js

```diff
@@ -8,13 +8,24 @@ import { noop } from 'lodash';
  * WordPress dependencies
  */
 import { forwardRef } from '@wordpress/element';
+import { isRTL } from '@wordpress/i18n';
+import { UP, RIGHT, DOWN, LEFT } from '@wordpress/keycodes';
 
 /**
  * Internal dependencies
  */
 import { InputRange as BaseInputRange } from './styles/range-control-styles';
 import { useDebouncedHoverInteraction } from './utils';
-import { useJumpStep } from '../utils/hooks';
```

Comment:
Since `keydown` events needed to be handled directly it meant the global handlers added by this hook are redundant. Everything else the hook does is trivial to implement.

Reply:
restored use of jump step hook in 0b8034e